packages/components/src/Filter/FilterMultiSelect/subcomponents/SearchInput/SearchInput.tsx

```diff
@@ -25,13 +27,24 @@ export const SearchInput = ({
   const reactId = useId()
   const inputId = id ?? reactId
 
+  const defaultAriaLabel = formatMessage({
+    id: "filterMultiSelectSearchInput.label",
+    defaultMessage: "Filter options by search query",
+    description: "Label for the search input",
+  })
+  const placeholder = formatMessage({
+    id: "filterMultiSelectSearchInput.placeholder",
+    defaultMessage: "Search…",
+    description: "Placeholder for the search input",
```

Comment:
Do you think this will be enough for the translation context? I always feel like with less content that's being translated we should provide more context for it - then again, in the grand scheme of this is a placeholder text and it is a generic prompt, sooo maybe give it a crack and see if they respond needing more context :)

Reply:
No idea tbh. Happy to change it if you can think of something more detailed to put. I couldn't think of how else to describe it though 😂

Reply:
Little verbose but maybe something like "Placeholder text for a search input to prompt users to enter a search term"? Not essential maybe lets add it and see if any issues arise